### ISSUE TITLE
Error : SSL CA bundle not found: 1 when connect to odoo

### ISSUE THREAD

**Issue by zeffrypfs** [obuchmann/odoo-jsonrpc#19]
## Reproduce Code
```
<?php

namespace Webkul\PrimaFoodCustom\Repositories;

use Obuchmann\OdooJsonRpc\Odoo;
use Obuchmann\OdooJsonRpc\Odoo\Request\Request;
use Webkul\Core\Eloquent\Repository;

class OdooRepository extends Repository
{
    protected $odoo;

    public function __construct()
    {
        $host = env('ODOO_HOST');
        $username = env('ODOO_USERNAME');
        $password = env('ODOO_PASSWORD');
        $database = env('ODOO_DATABASE');

        $odoo = new Odoo(new Odoo\Config(
            $host,
            $database,
            $username,
            $password,
            true
        ));


        $this->odoo = $odoo;

        // Connect to Odoo
        $this->odoo->connect();
    }

    public function createSaleOrder(SaleOrderCreateDto $saleOrderCreateDto)
    {
        try {
            $mappingOrderLine = array_map(function ($orderLine) {
                return [
                    0,
                    0,
                    [
                        'product_id' => $orderLine['product_id'], // 43826
                        'product_uom_qty' => $orderLine['product_uom_qty'], // 1
                        'price_unit' => $orderLine['price_unit'], // 6306.000
                        'warehouse_id' => $orderLine['warehouse_id'] // 94
                    ]
                ];
            }, $saleOrderCreateDto->order_line);

            $bodyCreate = [
                'partner_id' => $saleOrderCreateDto->partner_id, // 13982
                'client_order_ref' => $saleOrderCreateDto->client_order_ref, // @TEST-ECOMMERCE-0000001
                'partner_shipping_id' => $saleOrderCreateDto->partner_shipping_id, // 24442
                'x_studio_expected_receive_date' => $saleOrderCreateDto->x_studio_expected_receive_date, // 8/22/2024
                'commitment_date' => $saleOrderCreateDto->commitment_date, // "8/23/2024 12:00:00 AM"
                'analytic_account_id' => $saleOrderCreateDto->analytic_account_id, // 167
                'order_line' => $mappingOrderLine
            ];

            $saleOrderId = $this->odoo->model('sale.order')->create($bodyCreate);

            $confirmSaleOrderRequest = new ConfirmSaleOrderRequest($saleOrderId);
            $confirm = $this->odoo->execute($confirmSaleOrderRequest);

            return $confirm;
        } catch (\Throwable $th) {
            throw $th;
        }
    }

    /**
     * Specify Model class name
     *
     * @return mixed
     */
    function model()
    {
        return 'Webkul\PrimaFoodCustom\Contracts\Odoo';
    }
}

class SaleOrderCreateDto
{
    public function __construct(
        public int $partner_id,
        public string $client_order_ref,
        public int $partner_shipping_id,
        public string $x_studio_expected_receive_date,
        public string $commitment_date,
        public int $analytic_account_id,
        public array $order_line
    ) {}
}


class ConfirmSaleOrderRequest extends Request
{
    public function __construct(private int $id)
    {
        parent::__construct('sale.order', 'action_confirm');
    }

    public function toArray(): array
    {
        return [
            $this->id
        ];
    }
}

```

## Log 
```
{
    "message": "SSL CA bundle not found: 1",
    "exception": "InvalidArgumentException",
    "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\Handler\\CurlFactory.php",
    "line": 449,
    "trace": [
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\Handler\\CurlFactory.php",
            "line": 70,
            "function": "applyHandlerOptions",
            "class": "GuzzleHttp\\Handler\\CurlFactory",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\Handler\\CurlHandler.php",
            "line": 43,
            "function": "create",
            "class": "GuzzleHttp\\Handler\\CurlFactory",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\Handler\\Proxy.php",
            "line": 28,
            "function": "__invoke",
            "class": "GuzzleHttp\\Handler\\CurlHandler",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\Handler\\Proxy.php",
            "line": 48,
            "function": "GuzzleHttp\\Handler\\{closure}",
            "class": "GuzzleHttp\\Handler\\Proxy",
            "type": "::"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\PrepareBodyMiddleware.php",
            "line": 64,
            "function": "GuzzleHttp\\Handler\\{closure}",
            "class": "GuzzleHttp\\Handler\\Proxy",
            "type": "::"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\Middleware.php",
            "line": 31,
            "function": "__invoke",
            "class": "GuzzleHttp\\PrepareBodyMiddleware",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\RedirectMiddleware.php",
            "line": 71,
            "function": "GuzzleHttp\\{closure}",
            "class": "GuzzleHttp\\Middleware",
            "type": "::"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\Middleware.php",
            "line": 66,
            "function": "__invoke",
            "class": "GuzzleHttp\\RedirectMiddleware",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\HandlerStack.php",
            "line": 75,
            "function": "GuzzleHttp\\{closure}",
            "class": "GuzzleHttp\\Middleware",
            "type": "::"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\Client.php",
            "line": 333,
            "function": "__invoke",
            "class": "GuzzleHttp\\HandlerStack",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\Client.php",
            "line": 169,
            "function": "transfer",
            "class": "GuzzleHttp\\Client",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\guzzlehttp\\guzzle\\src\\Client.php",
            "line": 189,
            "function": "requestAsync",
            "class": "GuzzleHttp\\Client",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\obuchmann\\odoo-jsonrpc\\src\\JsonRpc\\Client.php",
            "line": 32,
            "function": "request",
            "class": "GuzzleHttp\\Client",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\obuchmann\\odoo-jsonrpc\\src\\Odoo\\Endpoint\\CommonEndpoint.php",
            "line": 19,
            "function": "__call",
            "class": "Obuchmann\\OdooJsonRpc\\JsonRpc\\Client",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\obuchmann\\odoo-jsonrpc\\src\\Odoo.php",
            "line": 57,
            "function": "authenticate",
            "class": "Obuchmann\\OdooJsonRpc\\Odoo\\Endpoint\\CommonEndpoint",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\obuchmann\\odoo-jsonrpc\\src\\Odoo.php",
            "line": 64,
            "function": "authenticate",
            "class": "Obuchmann\\OdooJsonRpc\\Odoo",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\packages\\Webkul\\PrimaFoodCustom\\src\\Repositories\\OdooRepository.php",
            "line": 32,
            "function": "connect",
            "class": "Obuchmann\\OdooJsonRpc\\Odoo",
            "type": "->"
        },
        {
            "function": "__construct",
            "class": "Webkul\\PrimaFoodCustom\\Repositories\\OdooRepository",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 952,
            "function": "newInstanceArgs",
            "class": "ReflectionClass",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 795,
            "function": "build",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php",
            "line": 963,
            "function": "resolve",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 731,
            "function": "resolve",
            "class": "Illuminate\\Foundation\\Application",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php",
            "line": 948,
            "function": "make",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 1066,
            "function": "make",
            "class": "Illuminate\\Foundation\\Application",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 982,
            "function": "resolveClass",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 943,
            "function": "resolveDependencies",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 795,
            "function": "build",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php",
            "line": 963,
            "function": "resolve",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 731,
            "function": "resolve",
            "class": "Illuminate\\Foundation\\Application",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php",
            "line": 948,
            "function": "make",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 1066,
            "function": "make",
            "class": "Illuminate\\Foundation\\Application",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 982,
            "function": "resolveClass",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 943,
            "function": "resolveDependencies",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 795,
            "function": "build",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php",
            "line": 963,
            "function": "resolve",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php",
            "line": 731,
            "function": "resolve",
            "class": "Illuminate\\Foundation\\Application",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php",
            "line": 948,
            "function": "make",
            "class": "Illuminate\\Container\\Container",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php",
            "line": 278,
            "function": "make",
            "class": "Illuminate\\Foundation\\Application",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php",
            "line": 1104,
            "function": "getController",
            "class": "Illuminate\\Routing\\Route",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php",
            "line": 1035,
            "function": "controllerMiddleware",
            "class": "Illuminate\\Routing\\Route",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php",
            "line": 818,
            "function": "gatherMiddleware",
            "class": "Illuminate\\Routing\\Route",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php",
            "line": 800,
            "function": "gatherRouteMiddleware",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php",
            "line": 784,
            "function": "runRouteWithinStack",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php",
            "line": 748,
            "function": "runRoute",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php",
            "line": 737,
            "function": "dispatchToRoute",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php",
            "line": 200,
            "function": "dispatch",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 144,
            "function": "Illuminate\\Foundation\\Http\\{closure}",
            "class": "Illuminate\\Foundation\\Http\\Kernel",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\barryvdh\\laravel-debugbar\\src\\Middleware\\InjectDebugbar.php",
            "line": 66,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 183,
            "function": "handle",
            "class": "Barryvdh\\Debugbar\\Middleware\\InjectDebugbar",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\packages\\Webkul\\Installer\\src\\Http\\Middleware\\CanInstall.php",
            "line": 30,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 183,
            "function": "handle",
            "class": "Webkul\\Installer\\Http\\Middleware\\CanInstall",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\packages\\Webkul\\Core\\src\\Http\\Middleware\\SecureHeaders.php",
            "line": 29,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 183,
            "function": "handle",
            "class": "Webkul\\Core\\Http\\Middleware\\SecureHeaders",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php",
            "line": 21,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TrimStrings.php",
            "line": 40,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 183,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Middleware\\TrimStrings",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize.php",
            "line": 27,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 183,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\packages\\Webkul\\Core\\src\\Http\\Middleware\\CheckForMaintenanceMode.php",
            "line": 93,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 183,
            "function": "handle",
            "class": "Webkul\\Core\\Http\\Middleware\\CheckForMaintenanceMode",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Http\\Middleware\\HandleCors.php",
            "line": 62,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 183,
            "function": "handle",
            "class": "Illuminate\\Http\\Middleware\\HandleCors",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Http\\Middleware\\TrustProxies.php",
            "line": 39,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 183,
            "function": "handle",
            "class": "Illuminate\\Http\\Middleware\\TrustProxies",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 119,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php",
            "line": 175,
            "function": "then",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php",
            "line": 144,
            "function": "sendRequestThroughRouter",
            "class": "Illuminate\\Foundation\\Http\\Kernel",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\public\\index.php",
            "line": 51,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Kernel",
            "type": "->"
        },
        {
            "file": "C:\\Project Prima Food Solutions\\php\\pfs-ecommerce-bagisto\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\resources\\server.php",
            "line": 16,
            "function": "require_once"
        }
    ]
}
```

**Comment by zeffrypfs** [comment]
I notice this bug because in **Config.php** especially at **getSslVerify** return type not sync with the [constructor](https://github.com/obuchmann/odoo-jsonrpc/blob/c1bef61c34faf26bcb5579230d832db1bb29065d/src/Odoo/Config.php#L57)

**Comment by vheins** [comment]
sorry for the mistake, @obuchmann  can you merge this pull request #20

**Comment by zeffrypfs** [comment]
@vheins I can't merge [this](https://github.com/obuchmann/odoo-jsonrpc/pull/20)

**Comment by vheins** [comment]
Is the problem not from that?

**Comment by zeffrypfs** [comment]
I mean I don't have permission for merge.

**Comment by obuchmann** [comment]
Merged an released as v1.7.1

Thank you for your work on this library!
